.gitignore

```diff
@@ -18,3 +18,6 @@
 
 # GitHub pages site generated by MkDocs
 /site
+
+#System
+.DS_Store
```

Comment:
Please move this directive at the top of file, next to InteliJ IDE ignored directory.